core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -303,6 +303,7 @@ public synchronized void losePrimacy() {
     } catch (IOException e) {
       LOG.warn("Error closing journal writer: {}", e.toString());
     } finally {
+      mAsyncJournalWriter.get().close();
       mAsyncJournalWriter.set(null);
       mRaftJournalWriter = null;
```

Comment:
Do we also need to close the `mRaftJournalWriter`?

Reply:
mRaftJournalWriter was closed above within `try` statement. I actually just moved closing async writer before that statement.